packages/live-status-gateway/src/collections/adLibActions.ts

```diff
@@ -0,0 +1,72 @@
+import { Logger } from 'winston'
+import { CoreHandler } from '../coreHandler'
+import { CollectionBase, Collection, CollectionObserver } from '../wsHandler'
+import { CoreConnection } from '@sofie-automation/server-core-integration'
+import { AdLibAction } from '@sofie-automation/corelib/dist/dataModel/AdlibAction'
+import { DBPartInstance } from '@sofie-automation/corelib/dist/dataModel/PartInstance'
+import { unprotectString } from '@sofie-automation/shared-lib/dist/lib/protectedString'
+import { PartInstanceName } from './partInstances'
+
+export class AdLibActionsHandler
+	extends CollectionBase<AdLibAction[]>
+	implements Collection<AdLibAction[]>, CollectionObserver<Map<PartInstanceName, DBPartInstance | undefined>>
+{
+	public observerName: string
+	private _core: CoreConnection
+	private _curRundownId: string | undefined
+
+	constructor(logger: Logger, coreHandler: CoreHandler) {
+		super('AdLibActionHandler', 'adLibActions', logger, coreHandler)
```

Comment:
Should these be using the `CollectionName` enum from `@sofie-automation/corelib/dist/dataModel/Collections` ?